task8/ServletsOnMaven/src/main/webapp/resources/js/SendRequests.js

```diff
@@ -0,0 +1,26 @@
+const headers = {
+    'Content-Type': 'application/json;charset=utf-8'
+};
+
+function sendGetRequest(url, paramName = null, id = null) {
```

Comment:
It's better have not just single param in the query, but to have possibility to pass (key, value) pairs (e.g. as an object) to add them to search query, cause it supports multiple params